modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,114 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify terms from a related custom object (child) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromCustomObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given two custom objects, A and B, each with a text field and publish them") {
+			for (var letters : list "A,B") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = "Custom Object ${letters} 182605",
+					objectName = "CustomObject${letters}182605",
+					pluralLabelName = "Custom Objects ${letters} 182605");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Field Text ${letters}",
```

Comment:
Why are we using a unique name for the field? Unless we have a specific reason for using a unique name, we can just name it `Text Field`